code/modules/clothing/spacesuits/syndi.dm

```diff
@@ -4,7 +4,8 @@
 	desc = "Top secret Spess Helmet."
 	icon_state = "syndicate"
 	item_state = "syndicate"
-	desc = "Has a tag: Totally not property of an enemy corporation, honest."
+	desc = "Has a tag on it: Totally not property of of a hostile corporation, honest!"
+	w_class = WEIGHT_CLASS_NORMAL

```

Comment:
You added a weight class that wasn't there previously, this should be stated in your PR.